packages/@aws-cdk/aws-redshift/test/integ.database.expected.json

```diff
@@ -1131,18 +1163,23 @@
             },
             {
               "Action": [
-                "secretsmanager:DescribeSecret",
-                "secretsmanager:GetSecretValue"
+                "secretsmanager:GetSecretValue",
+                "secretsmanager:DescribeSecret"
               ],
               "Effect": "Allow",
-              "Resource": [
-                {
-                  "Ref": "ClusterSecretAttachment769E6258"
-                },
-                {
-                  "Ref": "UserSecretAttachment02022609"
-                }
-              ]
+              "Resource": {
+                "Ref": "ClusterSecretAttachment769E6258"
+              }
+            },
+            {
+              "Action": [
+                "secretsmanager:GetSecretValue",
+                "secretsmanager:DescribeSecret"
+              ],
+              "Effect": "Allow",
+              "Resource": {
+                "Ref": "UserSecretAttachment02022609"
+              }
             }
```

Comment:
Unrelated integ test change here?

Reply:
updated the integ test for the millionth time. No idea why this keeps getting out of sync.